src/system/RoutesModule/Resources/views/Route/view.tpl

```diff
@@ -8,39 +8,45 @@
 {if count($items) > 1}
 {pageaddvar value='jquery-ui' name='javascript'}
 <script type="text/javascript">
-    jQuery(function(){
-        // Return a helper with preserved width of cells
-        var fixHelper = function(e, ui) {
-            ui.children().each(function() {
-                jQuery(this).css({width: jQuery(this).width()});
-            });
-            return ui;
-        };
+    /* <![CDATA[ */
+    ( function() {
```

Comment:
If you wrap it into an anonymus function, you could have also replaced `jQuery` by `$`.
```js
(function ($) {
    // ...
})(jQuery)
```